dev-app/src/screens/CollectCardPaymentScreen.tsx

```diff
@@ -162,10 +175,7 @@ export default function CollectCardPaymentScreen() {
         },
       ],
     });
-    const paymentMethods = ['card_present'];
-    if (enableInterac) {
-      paymentMethods.push('interac_present');
-    }
+    const paymentMethods = getEnabledPaymentMethodTypes(paymentMethodTypes);
```

Comment:
Nit: Not sure if it make more sense to change `getEnabledPaymentMethodTypes` to `getEnabledPaymentMethods` to align the output type/name not the input.